map/lanelet2_extension/test/src/test_route_checker.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "lanelet2_extension/utility/message_conversion.hpp"
+#include "lanelet2_extension/utility/route_checker.hpp"
+
+#include <autoware_auto_mapping_msgs/msg/had_map_bin.hpp>
+#include <autoware_auto_planning_msgs/msg/had_map_route.hpp>
+
+#include <gtest/gtest.h>
+
+using lanelet::Lanelet;
+using lanelet::LineString3d;
+using lanelet::Point3d;
+using lanelet::utils::getId;
+
+class TestSuite : public ::testing::Test
+{
+public:
+  TestSuite() : sample_map_ptr(new lanelet::LaneletMap())
+  {
+    // NOLINT
```

Comment:
Why NOLINT? :thinking:

Reply:
Please remove as many NOLINT as possible.
If you have to keep some NOLINT, please write the reason.

Reply:
Ok, rm NOLINT.
https://github.com/autowarefoundation/autoware.universe/pull/1149/commits/ec5cac09671784fe5d4b2dd03f25f1d67937d836